### PR TITLE
Add maven building to java image

### PR DESCRIPTION
**Pull request by waldman**

### CODE REVIEW

File: java/1.8/Dockerfile [hunk 1]
```diff
@@ -19,6 +19,15 @@ RUN cd /usr/lib && \
   sbt about && \
   rm -f /tmp/sbt.zip
 
+ENV MAVEN_VERSION 3.6.2
+ENV MAVEN_HOME /usr/lib/apache-maven-$MAVEN_VERSION
+ADD http://mirror.reverse.net/pub/apache/maven/maven-3/3.6.2/binaries/apache-maven-$MAVEN_VERSION-bin.zip /tmp/maven.zip
```
Comment (kimvsparrow) [inline java/1.8/Dockerfile]: there's another 3.6.2 to replace with `$MAVEN_VERSION` in this string

Comment (waldman) [reply java/1.8/Dockerfile]: OH! Indeed!

File: java/1.8/Dockerfile [hunk 2]
```diff
@@ -19,6 +19,15 @@ RUN cd /usr/lib && \
   sbt about && \
   rm -f /tmp/sbt.zip
 
+ENV MAVEN_VERSION 3.6.2
+ENV MAVEN_HOME /usr/lib/apache-maven-$MAVEN_VERSION
+ADD http://mirror.reverse.net/pub/apache/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.zip /tmp/maven.zip
```
Comment (kimvsparrow) [inline java/1.8/Dockerfile:24]: For string interpolation, do you need curly braces?
https://docs.docker.com/engine/reference/builder/#environment-replacement

Comment (waldman) [reply java/1.8/Dockerfile:24]: In bash/shell scripting it's more a kind of best-practice than a rule.

In this case the code was tested and is working fine as is (I also want to keep the same pattern as the other build systems installed).

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## [1.0.3] - 2019-10-30
+### Add
+- Add maven building to java image
```
Comment (mattgimbal) [inline CHANGELOG.md]: Can change this to:
```suggestion
- Maven building to java image
```

**Review by mattgimbal — APPROVED** [review]

**Review by kimvsparrow — APPROVED** [review]
👍

**Pull request merged**